src/client/components/wavesurfer/waveSurferApp.js

```diff
@@ -82,6 +82,15 @@ class WaveSurferApp {
   playPause() {
     this.waveSurfer.playPause();
     store.dispatch(playPause(this.waveSurfer.isPlaying()));
+    if (!this.waveSurfer.isPlaying()) {
+      console.log("update Time");
```

Comment:
Please don't useless `console.log` here.
Besides, maybe you can add some comment for why only setTime when not playing.